rali/rali/source/rali_api_meta_data.cpp

```diff
@@ -141,62 +152,88 @@ RALI_API_CALL raliGetImageLabels(RaliContext p_context, int* buf)
 }
 
 unsigned
-RALI_API_CALL raliGetBoundingBoxCount(RaliContext p_context, unsigned image_idx )
+RALI_API_CALL raliGetBoundingBoxCount(RaliContext p_context, int* buf)
 {
+    unsigned size = 0;
```

Comment:
Codacy found an issue: [The scope of the variable 'size' can be reduced.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6302037)